### PR TITLE
 update bazel-toolchains and add rbe_autoconfig target

### PR DESCRIPTION
**Pull request by nlopezgi**
For now we will have the digests here, but left a TODO to move them to some pin file (probably in bazel-toolchains repo) so that its easier to update.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -278,11 +278,21 @@ load(
...
+  name = "bazel_toolchains",
+  sha256 = "4b1468b254a572dbe134cc1fd7c6eab1618a72acd339749ea343bd8f55c3b7eb",
+  strip_prefix = "bazel-toolchains-d665ccfa3e9c90fa789671bf4ef5f7c19c5715c4",
+  urls = [
+    "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/d665ccfa3e9c90fa789671bf4ef5f7c19c5715c4.tar.gz",
+    "https://github.com/bazelbuild/bazel-toolchains/archive/d665ccfa3e9c90fa789671bf4ef5f7c19c5715c4.tar.gz",
+  ],
+)
+
+# TODO(nlopezgi): use pinned versions for digests once the container is made public
```
Comment (xingao267) [inline WORKSPACE]: use versions from a pin file once the container ....

Comment (nlopezgi) [reply WORKSPACE]: fixed

File: .bazelci/presubmit.yml [hunk 2]
```diff
@@ -63,10 +63,15 @@ platforms:
...
+    - "-//testdata:pkg_control_xz"
+    - "-//tests/docker:deb_image_with_dpkgs"
+    - "-//tests/docker:deb_image_with_dpkgs_test"
+    - "-//tests/docker:test_digest_output1"
     build_flags:
-    - "--extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:nosla_xenial_docker"
-    - "--host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:nosla_xenial_docker"
-    - "--platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:nosla_xenial_docker"
+    - "--extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.2:nosla_xenial_docker"
+    - "--host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/1.2:nosla_xenial_docker"
```
Comment (xingao267) [inline .bazelci/presubmit.yml]: You can probably use the latest docker_platform and toolchain alias here as well https://github.com/bazelbuild/bazel-toolchains/blob/master/.bazelci/presubmit.yml#L20

File: WORKSPACE [hunk 3]
```diff
@@ -279,10 +279,23 @@ _nodejs_image_repos()
...
+    strip_prefix = "bazel-toolchains-d665ccfa3e9c90fa789671bf4ef5f7c19c5715c4",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/ad7f0157e13d5d3a4b893652a375b03ee9e032b4.tar.gz",
-        "https://github.com/bazelbuild/bazel-toolchains/archive/ad7f0157e13d5d3a4b893652a375b03ee9e032b4.tar.gz",
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/d665ccfa3e9c90fa789671bf4ef5f7c19c5715c4.tar.gz",
+        "https://github.com/bazelbuild/bazel-toolchains/archive/d665ccfa3e9c90fa789671bf4ef5f7c19c5715c4.tar.gz",
     ],
 )
+
+load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
```
Comment (xingao267) [inline WORKSPACE]: load statements should be at top

File: .bazelci/presubmit.yml (hunk 2 above)
Comment (nlopezgi) [reply .bazelci/presubmit.yml]: fixed

File: WORKSPACE (hunk 3 above)
Comment (nlopezgi) [reply WORKSPACE]: Not in WORKSPACE; I dont think you can load before the http_archive rule that imports bazel_toolchains?

Comment (xingao267) [reply WORKSPACE]: got it

**Pull request merged**